source/documentation/common/security/proc-Applying_the_DISA_STIG_Security_Profile_RHV.adoc

```diff
@@ -19,16 +22,9 @@ image::common/images/security-policy.png[Security Policy]
 
 . On the *Security Policy* screen that opens, toggle the *Apply security policy* setting to *On*.
 
-. Select *[DRAFT] DISA STIG for Red Hat Virtualization Host (RHVH)*.
+. Select *DISA STIG for Red Hat Enterprise Linux 8*.
```

Comment:
worth an intro sentence saying that we use the standard RHEL profile

Reply:
Done